FinniversKit/Sources/Components/KeyValueGridView/KeyValuePair.swift

```diff
@@ -1,11 +1,32 @@
 public struct KeyValuePair: Hashable {
-    let title: String
-    let value: String
-    let accessibilityLabel: String?
+    public let title: String
+    public let value: String
+    public let accessibilityLabel: String?
+    public let valueStyle: Style?
 
-    public init(title: String, value: String, accessibilityLabel: String? = nil) {
+    public init(
+        title: String,
+        value: String,
+        accessibilityLabel: String? = nil,
```

Comment:
Are we supposed to get this from backend? Could we potentially default to something rather than nil?